### DISCUSSION TITLE
100% disk usage

### DISCUSSION THREAD

**Discussion by AhmedElhosieny** [Atlas-OS/Atlas#1282] [Help & Support]
sometimes the windows starts running slow so i open task manger and i see the disk usage is 100 put when i click on it it doesnt show what is using the disk so much

**Comment by meetox80** [comment]
is it a hdd?

**Reply by AhmedElhosieny** [reply]
yeah it is

**Comment by mian196** [comment]
maybe check disk for errors with chkdsk

**Reply by AhmedElhosieny** [reply]
found no problems

**Reply by mian196** [reply]
> found no problems

then check what program is taking full disk access maybe that can help
